src/hardware/siglent-sds/api.c

```diff
@@ -888,7 +888,7 @@ static int dev_acquisition_start(const struct sr_dev_inst *sdi)
 		break;
 	}
 
-	sr_scpi_source_add(sdi->session, scpi, G_IO_IN, 7000,
+	sr_scpi_source_add(sdi->session, scpi, G_IO_IN, 10,
```

Comment:
7000 is pretty big compared to other drivers. 10 seems to work OK with the X-E, but not sure about other series.